solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -49,7 +49,7 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
     using SafeERC20Ex for IERC20;
     using MathEx for *;
 
-    struct ProtectedLiquidity {
+    struct Position {
```

Comment:
Might be good to explicitly describe what `Position` is as it's less understandable than ProtectedLiquidy.

Reply:
It's probably as obscure as `ProtectedLiquidy` if you don't analyze the context (but in this case it could've been also `FooBar` 😅 )